perllib/FixMyStreet/TestMech.pm

```diff
@@ -653,6 +653,7 @@ sub create_body_ok {
     my ( $self, $area_id, $name, $params, $extra ) = @_;
 
     $params->{name} = $name;
+    $params->{cobrand} = delete $extra->{cobrand} if $extra->{cobrand};
```

Comment:
Is this solely so all the `create_body_ok` lines across all the `.t` files don't need updating or does it do something else? If not, perhaps updating all those lines would be a better approach. I think they all have at least an empty `{}` anyway so hopefully it'd just be the removal of a bunch of `}, {` instances.

Reply:
No, was just so I didn't have to update all the files :) Yeah, guess so, I imagine in most cases that's all the extra was for.